app/presenters/widget_presenter.rb

```diff
@@ -20,84 +20,71 @@ def render_partial
                                  :layout => false, :locals => {:presenter => self}).html_safe
   end
 
-  def widget_html_options(widget_content_type)
-    if widget_content_type == "chart"
-      {:title            => _("Open the chart and full report in a new window"),
-       :data_miq_confirm => _("This will show the chart and the entire report " \
-                              "(all rows) in your browser. Do you want to proceed?")}
-    else
-      {:title            => _("Open the full report in a new window"),
-       :data_miq_confirm => _("This will show the entire report (all rows) in your browser. " \
-                              "Do you want to proceed?")}
+  def widget_buttons
+    buttons = []
+    if role_allows?(:feature => "dashboard_add")
+      unless @sb[:dashboards][@sb[:active_db]][:locked]
+        buttons.push(:id         => "w_#{@widget.id}_close",
+                     :title      => _("Remove from Dashboard"),
+                     :name       => _(" Remove Widget"),
```

Comment:
extra leading space in `" Remove Widget"`?

Reply:
... looks most of the `name` values have that - please remove it and make sure the component shows a reasonable space when both fonticon and name are used :).

(Most likely, adding a `ng-if` for fonticon and forcing `fa-fw` (fixed width) for the icon if it *is* there is what you want.)